src/coreclr/vm/clsload.cpp

```diff
@@ -719,6 +716,9 @@ void ClassLoader::GetClassValue(NameHandleTable nhTable,
             }
             _ASSERTE(pTable);
 
+            mdToken             mdEncloser;
+            BOOL isNested = IsNested(pName, &mdEncloser);
```

Comment:
`IsNested` is a bit involved. In some cases like R2R, we may not need `IsNested`, so it makes sense to move the call down to where we are sure we need it.